src/lando/pushlog/pushlog.py

```diff
@@ -133,9 +133,17 @@ def record_push(self) -> Optional[Push]:
             push.commits.add(commit)
 
         push.save()
+        self.push = push
+        self.is_recorded = True
+
         logger.info(f"Successfully saved {push}")
 
-        self.is_recorded = True
+        try:
+            # XXX: share this instance
+            notifier = PulseNotifier()
+            notifier.notify_push(push)
+        except Exception as exc:
```

Comment:
Are there any specific exceptions that could be anticipated here?

Reply:
Not really, the point is to prevent unexpected exceptions from failing the push https://github.com/mozilla-conduit/lando/pull/220#discussion_r2030352190